lib/paypro.js

```diff
@@ -124,25 +124,34 @@ PayPro.get = function(opts, cb) {
     var decodedDetails = BitcorePayPro.PaymentDetails.decode(serializedDetails);
     var pd = new BitcorePayPro(COIN);
     pd = pd.makePaymentDetails(decodedDetails);
+    
+    var network = pd.get('network') == 'test' ? 'testnet' : 'livenet';
 
     var outputs = pd.get('outputs');
-    if (outputs.length > 1)
-      return cb(new Error('Payment Protocol Error: Requests with more that one output are not supported'))
 
-    var output = outputs[0];
+    var mapOutputs = function(output){
 
-    var amount = output.get('amount').toNumber();
-    var network = pd.get('network') == 'test' ? 'testnet' : 'livenet';
+      var amount = output.get('amount').toNumber();
+
+      // We love payment protocol
+      var offset = output.get('script').offset;
+      var limit = output.get('script').limit;
+
+      // NOTE: For some reason output.script.buffer
+      // is only an ArrayBuffer
+      var buffer = new Buffer(new Uint8Array(output.get('script').buffer));
+      var scriptBuf = buffer.slice(offset, limit);
+      var addr = new bitcore.Address.fromScript(new bitcore.Script(scriptBuf), network);
+      return {addr: addr.toString(), amount: amount}
+    };
+
+    var amountReduce = function (a, b) {
+      return b['amount'] == null ? a : a + b['amount'];
+    };
 
-    // We love payment protocol
-    var offset = output.get('script').offset;
-    var limit = output.get('script').limit;
+    var outs = outputs.map(mapOutputs);
 
-    // NOTE: For some reason output.script.buffer
-    // is only an ArrayBuffer
-    var buffer = new Buffer(new Uint8Array(output.get('script').buffer));
-    var scriptBuf = buffer.slice(offset, limit);
-    var addr = new bitcore.Address.fromScript(new bitcore.Script(scriptBuf), network);
+    var totalAmount = outs.reduce(amountReduce, 0);
```

Comment:
`amountReduce` should be replaced by the function, used only once and only 1 line. It removed readability.

Reply:
To follow our guidelines and avoid any crash with old javascript engine.
```javascript
var totalAmount = outs.reduce(function onReduce(a, b) {
 // your code...
});
```

Reply:
Done